.gitlab-ci.yml

```diff
@@ -0,0 +1,13 @@
+stages:
+  - build
+  - push
+
+build:
+  stage: build
+  script:
+    - docker build -t gpii/version-updater .
```

Comment:
I'd suggest adding `-rm=true` so intermediate containers don't pile up over time.

Reply:
Good catch. Looks like it's on by default:

`      --rm                         Remove intermediate containers after a successful build (default true)`

But I added it anyway for explicitness.